.github/workflows/ci.yml

```
@@ -0,0 +1,22 @@
+name: Bloop on Windows
+on:
+  push:
+jobs:
+  unix:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest]
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
+      - uses: olafurpg/setup-scala@v4
```

```suggestion
      - uses: olafurpg/setup-scala@v5
```

Windows support was added in v5 https://github.com/olafurpg/setup-scala/releases/tag/v5

---

Sweet, thanks!